### DISCUSSION TITLE
Any timeline for planned pro features?

### DISCUSSION THREAD

**Discussion by ramarivera** [johnlindquist/kit#1459] [General]
Basically title.
I read the FAQ but it is not clear when planned means.

Thanks!

**Comment by johnlindquist** [comment]
I'm planning on starting the v3 beta next week. 

v3 will have "multiple windows" as a pro feature (multiple editors/terminals/etc) so you can manage multiple scripts/UIs all running at the same time.

The next on the list after is a script generator tool which will allow you to create/save/manage/share scripts from the website, then 1-click install them into the app. This will be maximum priority as it should be key to growing the Script Kit user base and help people discover all the features.

Then the screenshot tool, then the rest. I'm really excited for the future :)
